### PR TITLE
Feature/update comorbitidy adjustment june2020

### PR DESCRIPTION
**Pull request by e-lipman**
Changes in PR:
1) Add adjustment for obesity
2) Add more detail to, and update, methods tab for comorbidity adjustment

### CODE REVIEW

File: src/info_html.R [hunk 1]
```diff
@@ -54,13 +54,39 @@ renderMethodsHtml <- function() {
...
-              "and gender from this preprint by", 
-              tags$a("Caramelo et al (2020).", href = urls$caramelo_etal_2020)
-      )
+              tags$a("OpenSafely. ", href = urls$open_safely), "Mutually adjusted odds ratios ", 
+              "(adjusted for other underlying conditions as well as age and gender) are used when available. ",
+              "Otherwise, unadjusted odds ratios are used but the user's risk is adjusted only for the two highest risk conditions ",
+              "inputted by the user. ", "These adjustment odds ratios will be updated as more research becomes available.",
+              "The following describes the current risk adjustment procedure:",
+              
+              tags$ul(tags$li("To adjust hospitalization risk, we use mutually adjusted odds ratios from Killerby et al. ",
```
Comment (xindyhu) [inline src/info_html.R:70]: @e-lipman could you add a hyperlink to Killerby et al?

File: src/info_html.R [hunk 2]
```diff
@@ -54,13 +54,39 @@ renderMethodsHtml <- function() {
...
+                              "This paper does not report odds ratios for 'immunocompromised condition' or ", 
+                              "'other chronic disease', so for these two categories we use unadjusted odds ratios from CDCC ",
```
Comment (xindyhu) [inline src/info_html.R:72]: @e-lipman, by CDCC do you mean to say "CCDC", which stands for China CDC?

File: src/info_html.R [hunk 3]
```diff
@@ -54,13 +54,39 @@ renderMethodsHtml <- function() {
...
+                              "and only adjust for these two categories if no other conditions are inputted by the user. ",
+                              "When the reported ratio is below 1, we bring it up to 1, ",
+                              "reflecting the assumption that underlying medical conditions do not lessen suseptibility to COVID-19."),
+                      tags$li("To adjust for ICU risk, we use unadjusted odds ratios from CDCC (for all but obesity). ",
```
Comment (xindyhu) [inline src/info_html.R:76]: @e-lipman Similar as above, this should be CCDC right?

File: src/info_html.R [hunk 4]
```diff
@@ -54,13 +54,39 @@ renderMethodsHtml <- function() {
...
+                              "To avoid overinflating risk due to the odds ratios being unadjusted, we adjust only for ",
+                              "the two highest risk conditions inputted by the user. ",
+                              "To adjust for obesity, we use the mutually adjusted odds ratio from Simonnet et al."),
```
Comment (xindyhu) [inline src/info_html.R:79]: @e-lipman, please add a hyperlink to Simonnet et al.

File: src/info_html.R [hunk 5]
```diff
@@ -54,13 +54,39 @@ renderMethodsHtml <- function() {
...
+                      tags$li("To adjust death risk, we use mutually adjusted odds ratios from OpenSafely. ",
```
Comment (xindyhu) [inline src/info_html.R:80]: @e-lipman, please add a hyperlink to OpenSAFELY study. Note the capitalized and uncapitalized letters in the study name.

File: src/info_html.R [hunk 6]
```diff
@@ -54,13 +54,39 @@ renderMethodsHtml <- function() {
...
+                              "This paper does not report odds ratios for ", 
+                              "'other chronic disease', so for this one categoriy we use the unadjusted odds ratio from CDCC ",
```
Comment (xindyhu) [inline src/info_html.R:82]: CDCC -> CCDC

File: src/info_html.R [hunk 7]
```diff
@@ -54,13 +54,39 @@ renderMethodsHtml <- function() {
...
+                              "and only adjust for this category if no other conditions are inputted by the user. ",
+                              "When the reported ratio is below 1, we bring it up to 1, ",
+                              "reflecting the assumption that underlying medical conditions do not lessen suseptibility to COVID-19.",
+                              "The ratios from OpenSafely pertain to risk of death from COVID-19 in the entire population, ",
```
Comment (xindyhu) [inline src/info_html.R:86]: OpenSafely -> OpenSAFELY

File: src/helper_input.R [hunk 8]
```diff
@@ -61,6 +61,7 @@ collapseStory <- function() {
                              "Current or former smoker" = "is_smoker",
                              "Immunocompromised condition" = "is_immune",
                              "Chronic lung disease or asthma" = "is_lung",
+                             "Obesity" = "is_obesity",
```
Comment (xindyhu) [inline src/helper_input.R:64]: Let's change the label to "Obesity (BMI >=30kg/m2)" I don't know Rshiny's ability in rendering math equations and superscripts, but should be doable.

File: src/global_var.R [hunk 9]
```diff
@@ -42,15 +45,16 @@ death_list = c(0.00161, 0.00695, 0.0309,  0.0844,  0.161,   0.595,   1.93,   4.2
...
 
 renal_or    = c(2.6, 5.82, 1.72) # death OR not available in CCDC
 cvd_or      = c(1.4, 4.88, 1.27)
 diabetes_or = c(3.1, 4.57, 1.79)
-hyper_or    = c(1.1, 4.57, 1) # use the same as diabetes
+hyper_or    = c(1.1, 4.57, 1) # for ICU same as diabetes
 smoker_or   = c(2.3, 2.64, 1.12) # death OR not available in CCDC
 immune_or   = c(2.58, 2.86, 1.69) # death OR not available in CCDC
 lung_or     = c(1, 2.83, 1.78)
+obesity_or  = c(1.9, 3.41, 1.46)
```
Comment (xindyhu) [inline src/global_var.R:57]: I can't leave my review for the xlsx file above so I add it here.
On the "Odds ratio for app" tab, cell D10 should be 3.41

**Comment by e-lipman** [comment]
Hi Cindy,
I have made changes based on your comments. Let me know if I missed anything!
Best,
Erin

**Pull request merged**